tools/llm_bench/llm_bench_utils/ov_utils.py

```diff
@@ -279,6 +279,23 @@ def create_ldm_super_resolution_model(model_path, device, **kwargs):
     return ov_model, from_pretrained_time
 
 
+def create_genai_speech_2_txt_model(model_path, device, **kwargs):
+    import openvino_genai as ov_genai
+    if kwargs.get("genai", False) is False:
+        raise RuntimeError('==Failure the command line does not set --genai ==')
+    if is_genai_available(log_msg=True) is False:
+        raise RuntimeError('==Failure genai is not enable ==')
+    start = time.perf_counter()
+    genai_pipe = ov_genai.WhisperPipeline(
+        str(model_path), device=device.upper()
```

Comment:
```suggestion
        model_path, device.upper()
```

Reply:
python Pathlike objects are not supported as path to model in genai API as I know or something changed for this case?

Reply:
yes, in this PR https://github.com/openvinotoolkit/openvino.genai/pull/1031 where we have migrated to `std::filesystem::path` which allowed to support Path-like objects in Python via pybind11